src/cli/configuration_builder.ts

```diff
@@ -143,6 +143,7 @@ export default class ConfigurationBuilder {
 
   async expandFeaturePaths(featurePaths: string[]): Promise<string[]> {
     featurePaths = featurePaths.map((p) => p.replace(/(:\d+)*$/g, '')) // Strip line numbers
+    featurePaths = [...new Set(featurePaths)] // Deduplicate the feature files; // Deduplicate the feature files
```

Comment:
duplicate comment, second could be removed

Reply:
Oops. 😳